charts/kube-prometheus-stack/templates/prometheus/servicemonitors.yaml

```
@@ -21,6 +21,15 @@ items:
     {{- if .jobLabel }}
       jobLabel: {{ .jobLabel }}
     {{- end }}
+    {{- if .metricRelabelings }}
+          metricRelabelings:
+            {{ toYaml .metricRelabelings | nindent 12 }}
+    {{- end }}
+    {{- if .relabelings }}
+          relabelings:
+            {{ toYaml .relabelings | nindent 12 }}
```

```suggestion
            {{- toYaml .relabelings | nindent 12 }}
```

the hypen is important to strip the space before.

---

Added.